server/models/investment.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const investmentSchema = new mongoose.Schema({
```

Comment:
Any reasons not to reference the user and project schemas directly?

Reply:
I'm not sure if this is the right way to do it, but I'm referencing them on the other side, so the user can .populate all of their investments, and a project can .populate all of the investments made towards it.  line 59 in project model and line 38 in the user model.  This would create the references you're talking about right?